src/NuGet.Clients/NuGet.SolutionRestoreManager/SolutionRestoreWorker.cs

```diff
@@ -522,9 +522,6 @@ await _solutionLoadedEvent.WaitAsync()
                             {
                                 if (isAllProjectsNominated)
                                 {
-                                    // Counterfactual logging for bulk file restore coordination scenario
-                                    Common.TelemetryActivity.EmitTelemetryEvent(new CpsBulkFileRestoreCoordinationEvent());
```

Comment:
I believe you will also need to remove tests:

https://github.com/NuGet/NuGet.Client/blob/264f87bafa108e946a31b9d5e16951220dae7e84/test/NuGet.Clients.Tests/NuGet.VisualStudio.Common.Test/Telemetry/CpsBulkFileRestoreCoordinationEventTests.cs#L24

Reply:
yep, the build failed :D 
Fixed now.